src/Nethermind/Nethermind.Network.Discovery/Discv5/DiscoveryV5App.cs

```diff
@@ -33,11 +34,13 @@ public class DiscoveryV5App : IDiscoveryApp
     private readonly IApiWithNetwork _api;
     private readonly Logging.ILogger _logger;
     private readonly IDiscoveryConfig _discoveryConfig;
-    private readonly SimpleFilePublicKeyDb _discoveryDb;
+    private readonly IFullDb _discoveryDb;
```

Comment:
NP: Why not IKeyValueStore? missing some method?

Reply:
Some of the methods are missing from `IKeyValueStore`, `GetAllValues` being an example. Replaced it with `IDb` instead.